BackEnd/SurveyAPI/appsettings.json

```diff
@@ -5,5 +5,18 @@
       "Microsoft.AspNetCore": "Warning"
     }
   },
+  "service_config": {
```

Comment:
@omeryurd @tarunJeevan @saipavanchowdari Ideally, these would be setup on the environment hosting this api and wouldn't be here as they are supposed to be secret.. you may receive or have already received an alert from GitHub warning of a possible exposure.... you may ignore